qa/util/src/main/java/io/camunda/qa/util/cluster/TestWebappClient.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.qa.util.cluster;
+
+import io.camunda.authentication.config.WebSecurityConfig;
+import io.camunda.zeebe.util.Either;
+import java.io.IOException;
+import java.net.CookieManager;
+import java.net.HttpCookie;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.Builder;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.util.List;
+import java.util.function.Consumer;
+
+public class TestWebappClient {
+
+  private final URI endpoint;
+
+  public TestWebappClient(final URI endpoint) {
+    this.endpoint = endpoint;
+  }
+
+  public TestLoggedInWebappClient logIn(String username, String password) {
+
+    final var cookieManager = new CookieManager();
+    final var httpClient = HttpClient.newBuilder().cookieHandler(cookieManager).build();
+    final var loginUri = endpoint.resolve("login");
+
+    final var loginRequest =
+        HttpRequest.newBuilder()
+            .uri(loginUri)
+            .header("Content-Type", "application/x-www-form-urlencoded")
```

Comment:
Just some style nitpic, since in previous tests we used `HttpHeaders.*`, `MediaType.*`, maybe consider using constants here as well if library present in the module classpath.